Samples/WindowsCopilotRuntime/README.md

```diff
@@ -37,17 +37,12 @@ join the [Windows Insider Program](https://insider.windows.com).
 
 See [additional instructions](./cpp-console-unpackaged/README.md) for using [the C++ & CMake sample](./cpp-console-unpackaged/CMakeLists.txt).
 
-## Special Considerations for Debugging as an Unpackaged App in Visual Studio
-
-- This project is not designed to be fully functional in an unpackaged app. However, Windows Copilot Runtime does support unpackaged app.
-- To enable proper startup as an unpackaged app, you need to bootstrap the Windows App SDK either Programmatically or By adding the following configuration to the `.csproj` file during the build process:
- ```xml
-  <WindowsAppSdkBootstrapInitialize>true</WindowsAppSdkBootstrapInitialize>
-  <WindowsPackageType>None</WindowsPackageType>
- ```
-- Alternatively, if you are using the Developer Command Prompt for Visual Studio, you can run the app as an ARM64 version using the following command, with the bootstrap property provided:
+## Special Considerations for Unpackaged & Self-Contained Mode with WCR
+
+- This project is not intended to be fully functional when running as an unpackaged app. However, Windows Copilot Runtime (WCR) does support unpackaged apps.
+- Self-contained mode is fully supported by WCR APIs as well.
+- The following command demonstrates how to run the app as an ARM64 unpackaged application in self-contained mode (for both WinAppSDK and .NET):
```

Comment:
@pratikone I don't understand why the project properties were removed in favor of command line parameters, which would vary with build tool and aren't even available in VS.